src/main/resources/assets/computercraft/lua/rom/programs/advanced/multishell.lua

```diff
@@ -315,7 +315,15 @@ while #tProcesses > 0 do
     elseif sEvent == "mouse_drag" or sEvent == "mouse_up" or sEvent == "mouse_scroll" then
         -- Other mouse event
         local p1, x, y = tEventData[2], tEventData[3], tEventData[4]
-        if not (bShowMenu and y == 1) then
+        if bShowMenu and sEvent == "mouse_scroll" and y == 1 then
+            if p1 == -1 and nScrollPos ~= 1 then
+                nScrollPos = nScrollPos - 1
+                redrawMenu()
+            elseif bScrollRight and p1 == 1 then
+                nScrollPos = nScrollPos + 1
+                redrawMenu()
+            end
+        elseif not (bShowMenu and y == 1) then
```

Comment:
I feel like this would be clearer as `elseif not bShowMenu or y ~= 1 then`, the `and` in a `not` is a bit more confusing to read. Arguably you could just move `sEvent == "mouse_scroll"` into a separate `if` and make this just an `else` statement though.